styles/styles.css

```
@@ -0,0 +1,200 @@
+  font-display: swap;
+}
+
+body {
+  font-family: "PT Sans", sans-serif;
+  font-size: 18px;
+  line-height: 21px;
+  color: #000000;
+  background-color: #ffffff;
+}
+
+.page-header {
+  color: #000000;
+  background: #ffffff;
+}
+
+.navigation-link{
+  font-family: "PT Sans";
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 24px;
+  color: #000000;
+  text-decoration: none;
+  text-transform: capitalize;
+}
+
+.main-index {
+  color: #000000;
+  background-color: #ffffff;
+  background-image: url("../images/index-background.jpg");
+  background-repeat: no-repeat;
+}
+
+.main-index h2{
+  font-family: "PT Sans";
+  font-weight: 700;
+  font-size: 30px;
+  text-align: center;
+  line-height: 36px;
+  text-transform: uppercase;
+  background-color: #ffffff;
+}
+
+.advantages-title-city h3{
+  font-family: "PT Sans";
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 28px;
+  text-align: center;
+  color: #ffffff;
+  background-color: #82b3d3;
+  text-transform: uppercase;
+}
+
+.reasons{
+  font-family: "PT Sans";
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 26px;
+  color: #333333;
+  text-align: center;
+  background-color: #ffffff;
+}
+
+.main-text {
+  text-align: center;
+  color: #000000;
+  background-color: #ffffff;
+}
+
+.advantages{
+  font-size: 24px;
+  line-height: 28px;
+  text-align: center;
+}
+
+main h3 {
+  text-transform: uppercase;
+}
+
+.reserve {
+  font-family: "PT Sans";
+  font-style: normal;
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 36px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #000000;
+}
+
+.appointment-form {
+  text-align: center;
+}
+
+.services-description {
+  font-family: "PT Sans";
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 26px;
+  color: #333333;
+  text-align: center;
+}
+
+.services-name {
+  font-family: "PT Sans";
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 28px;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+.services-list {
+  text-align: center;
+}
+
+.contacts-phone {
+  font-family: "PT Sans";
+  font-style: normal;
+  font-size: 40px;
+  line-height: 40px;
+  font-weight: 400;
+  text-align: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000000;
+  background-color: #ffffff;
+}
+
+.hotels-card-title {
+  font-family: "PT Sans";
+  font-weight: 700;
+  font-style: normal;
+  font-size: 24px;
+  line-height: 28px;
+  color: #000000;
+}
+
+.hotels-list h4 {
+  font-family: "PT Sans";
+  font-weight: 400;
+  font-style: normal;
+  font-size: 18px;
+  line-height: 21px;
+  color: #333333;
+}
+
+.hotels-card-price {
+  font-family: "PT Sans";
+  font-weight: 400;
+  font-style: normal;
+  font-size: 18px;
+  line-height: 21px;
+  color: #333333;
+}
+
+.hotels-list button {
+
+}
+
+.hotels-card-link {
+  text-decoration: none;
+}
+
+.mailing-title{
```

```suggestion
.mailing-title {
```
Предлагаю везде сохранять одинаковый стиль - пробел после селектора